### PR TITLE
fix(fully_async): seed optimizer training steps before init

### LINKED ISSUE #6683: lr is always 0 in fully async
### System Info

main verl

### Information

- [ ] The official example scripts
- [x] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [x] My own task or dataset (give details below)

### Reproduction

Whether using FSDP or VeOmni, the training step is read from the optim config when building the learning rate scheduler.
| pic1 | pic2 |
|-----|-----|
| Image | Image |

The running script is modified based on the sync mode. Tasks started via main_ppo do not have this issue. 
Code inspection reveals that this branch is executed when creating the trainer under fully async mode：
`verl.workers.engine.veomni.transformer_impl.VeOmniEngine._build_lr_scheduler`
where will do：`train_steps=optim_config.total_training_steps`，the default value of optim_config.total_training_steps here is -1.

Tasks started through main_ppo assign trainer.total_training_steps to optim_config.total_training_steps：
Image

In Async mode, It can be observed that a sync about  total_train_steps between rollouter and trainer after create trainer. But yet this operation no longer takes effect on optim.lr at this point. If actor_rollout_ref.actor.optim.total_training_steps is not configured separately, the lr will remain 0 all the time.

Image

### Expected behavior

The optim_config.total_training_steps is automatically defined by the rollout.total_rollout_steps.

### PR DESCRIPTION
**Pull request by Epochex**
Fixes #6683.

In fully async mode the trainer workers build the optimizer and LR scheduler during `init_workers()`, but total training steps were only copied from the rollouter afterwards. When `actor.optim.total_training_steps` stayed at the default `-1`, the scheduler could be built with the wrong train step count.

This seeds actor/critic optimizer `total_training_steps` from `rollout.total_rollout_steps` before trainer worker initialization, while preserving an explicitly configured optimizer value.

Validation:
- `uv run --no-project --with ruff ruff check verl/experimental/fully_async_policy/fully_async_main.py tests/experimental/fully_async_policy/test_total_training_steps_on_cpu.py`
- `uv run --no-project --with ruff ruff format --check verl/experimental/fully_async_policy/fully_async_main.py tests/experimental/fully_async_policy/test_total_training_steps_on_cpu.py`
- `git diff --check`
- helper semantic smoke with OmegaConf passed

I could not run the standard pytest command on Windows because `requirements.txt` pulls `liger-kernel` -> `triton`, and `triton` has no `win_amd64` wheel. Linux CI should cover the normal test environment.

### CODE REVIEW

**Comment by wuxibin89** [comment]
Duplicated with https://github.com/verl-project/verl/pull/6684
